lib/components/BulletText/BulletText.tsx

```diff
@@ -1,10 +1,10 @@
 import cx from 'clsx';
-import React, { FunctionComponent, isValidElement, ReactElement } from 'react';
-import { Text } from '../Text';
+import React, { FunctionComponent, isValidElement, ReactNode } from 'react';
+import { Text } from '../Typography/Text';
 import styles from './style.scss';
 
 export interface IProps {
-	bullet?: string | ReactElement;
+	bullet?: ReactNode;
```

Comment:
Does this mean we're not going to be able to pass say a number as bullet anymore?

Reply:
`ReactNode` allows for "pretty much anything".

```ts
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;
type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```